meta-luneos/recipes-support/pidgin/whatsapp-purple.bb

```diff
@@ -0,0 +1,28 @@
+
+SUMMARY = "WhatsApp LibPurple Plugin"
+SECTION = "webos/services"
+LICENSE = "GPLv2.0+"
+LIC_FILES_CHKSUM = "file://debian/copyright;md5=6b066e36c765e83968e71478f54618f3"
+
+DEPENDS = "pidgin protobuf-native protobuf freeimage"
+
+SRCREV = "07ed931a3dd04c98a5f36e52129aa2cfe743a024"
+
+inherit pkgconfig
+
+SRC_URI = "git://github.com/davidgfnet/whatsapp-purple"
+PV = "0.9.0+git${SRCPV}"
+S = "${WORKDIR}/git"
+
+do_compile() {
+    oe_runmake CC="${CC}" CXX="${CXX}" EXTRA_INCLUDES="${TARGET_CFLAGS}" AR="${AR}";
+}
+
+do_install() {
+    oe_runmake DESTDIR="${D}" install;
+}
+
+FILES_${PN} += " \
+    ${libdir} \
+    ${datadir} \
```

Comment:
I get the following when I omit libdir and/or datadir:

WARNING: whatsapp-purple-0.9.0+gitAUTOINC+07ed931a3d-r0 do_package: QA Issue: whatsapp-purple: Files/directories were installed but not shipped in any package:
  /usr/lib/purple-2/libwhatsapp.so
Please set FILES such that these items are packaged. Alternatively if they are unneeded, avoid installing them or delete them within do_install.
whatsapp-purple: 1 installed and not shipped files. [installed-vs-shipped]